### PR TITLE
upt: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/upt.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# upt
+
+> Interfaz unificada para gestionar paquetes en varios sistemas operativos, tales como Windows, diversas distribuciones de Linux, macOS, FreeBSD e incluso Haiku.
+> Requiere que el gestor de paquetes nativo del sistema operativo esté instalado.
```
Comment (tricantivu) [inline pages.es/common/upt.md:4]:
```suggestion
> Requiere que el gestor de paquetes nativo del sistema operativo este instalado.
```

File: pages.es/common/upt.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+> Véase también: `flatpak`, `brew`, `scoop`, `apt`, `dnf`.
```
Comment (tricantivu) [inline pages.es/common/upt.md:5]:
```suggestion
> Vea también: `flatpak`, `brew`, `scoop`, `apt`, `dnf`.
```

File: pages.es/common/upt.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+> Más información: <https://github.com/sigoden/upt>.
+
+- Actualiza la lista de paquetes disponibles:
+
+`upt update`
+
+- Busca un paquete determinado:
```
Comment (tricantivu) [inline pages.es/common/upt.md:12]:
```suggestion
- Busca un paquete:
```

File: pages.es/common/upt.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+`upt search {{término_de_búsqueda}}`
+
+- Muestra información sobre un paquete:
+
+`upt info {{paquete}}`
+
+- Instala un paquete determinado:
```
Comment (tricantivu) [inline pages.es/common/upt.md:20]:
```suggestion
- Instala un paquete:
```

File: pages.es/common/upt.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+`upt install {{paquete}}`
+
+- Elimina un paquete determinado:
```
Comment (tricantivu) [inline pages.es/common/upt.md:24]:
```suggestion
- Elimina un paquete:
```

File: pages.es/common/upt.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+`upt {{remove|uninstall}} {{paquete}}`
+
+- Actualiza todos los paquetes instalados:
+
+`upt upgrade`
+
+- Actualiza un paquete determinado:
```
Comment (tricantivu) [inline pages.es/common/upt.md:32]:
```suggestion
- Actualiza un paquete:
```

File: pages.es/common/upt.md (hunk 1 above)
Comment (kant) [reply pages.es/common/upt.md:4]: In this case, it is a verb. So it should be with a "tilde"

Comment (tricantivu) [reply pages.es/common/upt.md:4]: You are probably right, my bad. I deleted the suggestion.

https://www.diccionariodedudas.com/este-este-o-este/

**Review by tricantivu — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**